.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,28 @@
+name: static analysis
+
+on:
+  push:
+    branches:
+      - master
+      - development
+        #    paths:
```

Comment:
Can we remove these (and the following) commented lines?

Reply:
I've now set it up such that it's run only if any of scripts have changed.